deployment/ansible/roles/configure/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+# Copyright 2022 Samsung Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: read genesis file
+  stat:
+    path: "{{ dcl_home }}/config/genesis.json"
+  register: genesis_file
+
+- name: initialize the layout
+  changed_when: false
+  command: "{{ dcld.path }} init '{{ moniker | default(ansible_hostname) }}' --chain-id '{{ chain_id }}' --home {{ dcl_home }}"
+  when: not genesis_file.stat.exists
+
+- name: setup TOMLs files
+  copy:
+    dest: "{{ dcl_home }}/config/{{ item.key }}.toml"
+    content: "{{ item.value | sivel.toiletwater.to_toml  }}"
```

Comment:
It seems like you are replacing the whole content of a config file here. But we should keep untouched the remaining ones.